packages/next-css/readme.md

```diff
@@ -195,6 +195,34 @@ Create a CSS file `style.css` the CSS here is using the css-variables postcss pl
 
 When `postcss.config.js` is not found `postcss-loader` will not be added and will not cause overhead.
 
+### Leverage long-term browser caching of style.css in production
```

Comment:
This should be the main installation step if we change the filename to include the hash.

Reply:
Did you mean that we should incorporate it into the "Usage" section instead?

Reply:
Yeah because otherwise people will create issues that the package doesn't work in production 😄

Reply:
Good point! Less confusion. Updated in 127db81. Please let me know what you think.